javascript/core/objects/converting-to-and-from-json.md

```diff
@@ -0,0 +1,78 @@
+---
+author: EwaTrzemzalska
+
+type: normal
+
+category: fundamental
```

Comment:
`category` field should be an array of one or more categories.

In YAML, this is represented as:

```js
category:
- fundamental
```